Sources/Application/SceneDelegate.swift

```diff
@@ -14,12 +14,14 @@ final class SceneDelegate: UIResponder,
 
     private var rootCoordinator: QualifyingCoordinator?
 
+    private let walletAvailibilityService = WalletAvailabilityService()
```

Comment:
```suggestion
    private let walletAvailabilityService = WalletAvailabilityService()
```

Reply:
Done, sorry about that, I copied it and used the same one across all my changes :sweat_smile: